### PR TITLE
AddDbDetails page is totally changed to the material UI

### PR DESCRIPTION
**Pull request by Manikandaselvan03**

### CODE REVIEW

File: acciom_ui/src/containers/TestSuiteUpload.js [hunk 1]
```diff
@@ -446,7 +446,7 @@ class TestSuiteUpload extends React.Component {
 					<h4 className='pageTitle update-data-profiling-title main_titles'>Update Data Profiling</h4>
 					<Button variant="contained" className="button-colors brows-btn" onClick={ (e) => handleSwitchProject()}>Switch Project</Button>
 				</div>
-				<Paper style={{'height':'200px','marginTop':'8px'}}>
+				<Paper style={{'height':'163px','marginTop':'3px'}}>
```
Comment (HakifRaseen) [inline acciom_ui/src/containers/TestSuiteUpload.js]: use classes

Comment (Manikandaselvan03) [reply acciom_ui/src/containers/TestSuiteUpload.js]: ok

File: acciom_ui/src/css/styles.css [hunk 2]
```diff
@@ -4256,7 +4256,7 @@ html
     padding-left:8px;
     font-weight: 200;
     margin-bottom: 0px !important;
-    margin-top: 14px;
+    /* margin-top: 14px; */
```
Comment (HakifRaseen) [inline acciom_ui/src/css/styles.css]: remove commented lines

File: acciom_ui/src/css/Db-ui-styles.css [hunk 3]
```diff
@@ -2120,13 +2131,13 @@ body,html{
     height: 31px;
     border: none !important;
     margin-left: 10px !important;
-    margin-top: 0px !important;
+    /* margin-top: 0px !important; */
```
Comment (HakifRaseen) [inline acciom_ui/src/css/Db-ui-styles.css]: remove comments

**Pull request merged**